AntonBabunin/src/main/java/ua/artcode/market/interfaces/IAppDb.java

```diff
@@ -0,0 +1,37 @@
+package ua.artcode.market.interfaces;
+
+import ua.artcode.market.models.Bill;
+import ua.artcode.market.models.Product;
+import ua.artcode.market.models.Salesman;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public interface IAppDb {
+
+    Map<Product, Integer> getProducts();
```

Comment:
Why do you use Map?

Reply:
I use Map here because it is a "DB" of products. What products are in the shop and quantity of each of them